File: backend/src/main/kotlin/com/moseoh/danggeunclone/auth/domain/Token.kt
package com.moseoh.danggeunclone.auth.domain

import jakarta.persistence.Entity
import jakarta.persistence.Id
import java.util.*

@Entity
data class Token(
    @Id
    val userId: Long,
    val accessToken: String,
    val refreshToken: String,
    val expiredAt: Date,
)

File: client/lib/models/token.dart
class Token {
  final String accessToken;
  final String refreshToken;

  Token(this.accessToken, this.refreshToken);

  Token.fromJson(Map<String, dynamic> json)
      : accessToken = json['accessToken'],
        refreshToken = json['refreshToken'];

  Map<String, dynamic> toJson() => {
        'accessToken': accessToken,
        'refreshToken': refreshToken,
      };
}
